lib/experimental/script-modules.php

```diff
@@ -197,3 +197,25 @@ function gutenberg_dequeue_module( $module_identifier ) {
 	_deprecated_function( __FUNCTION__, 'Gutenberg 17.6.0', 'wp_dequeue_script_module' );
 	wp_script_modules()->dequeue( $module_identifier );
 }
+
+add_action(
+	'init',
+	function () {
+		wp_register_script_module( '@wordpress/api-fetch', gutenberg_url( 'build/script-modules/apiFetch.min.js' ) );
+
+		// add_filter(
+		// 	'scriptmoduleconfig_@wordpress/api-fetch',
```

Comment:
This should be restored with the correct filter name:

https://github.com/WordPress/gutenberg/blob/5738e9b0529eae4678539e28d83747258824df5d/lib/experimental/script-modules.php#L246